commands/Icommand.py

```diff
@@ -0,0 +1,3 @@
+class ICommand():
```

Comment:
This class should be abstract 
https://www.geeksforgeeks.org/abstract-classes-in-python/